department-of-veterans-affairs/caseflow
# Allow VLJs to Assign Attorney Task to Themselves

Connects CASEFLOW-2203

### Description
This adjusts logic in `AttorneyTask` and `AssignToAttorneyWidget` to allow the VLJ to assign a case to themselves.

### Acceptance Criteria
- [ ] Current VLJ user's name appears in assignment dropdown in modal (from Case Details)
- [ ] Current VLJ user's name appears in assignment dropdown from bulk assign queue
- [ ] Assigning to VLJ is allowed

### Testing Plan
1. Log in as VLJ BVAAABSHIRE
2. Go to queue and switch to "Assign BVAAABBSHIRE team cases"
3. Click on one of the cases to go to Case Details page
4. Select "Assign to attorney"
5. Verify that VLJ's name is in dropdown and select it
6. Complete submission and verify no errors
7. Go back to "Assign BVAAABBSHIRE team cases" view
8. Select one or more cases, and assign them to self
9. Verify no errors

### User Facing Changes

 BEFORE|AFTER
 ---|---
![Screen Shot 2021-09-23 at 11 49 01](https://user-images.githubusercontent.com/2581274/134540779-47078f75-4731-41b4-9a8f-cb9b4a382b31.png) | ![Screen Shot 2021-09-23 at 11 49 28](https://user-images.githubusercontent.com/2581274/134540806-b7426636-2710-4c36-9148-5f90206f8391.png)
